backend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dotenv": "^16.4.7",
+    "express": "^4.21.2",
+    "pg": "^8.13.1"
+  }
```

Comment:
I have a suggestion, but I don't know if you will use it. If you want to add a testing framework, we can use one. For example, if you want to use jest, you will need to do the following:

1. Add this after the dependencies:
```json
 "devDependencies": {
    "jest": "^29.0.0" // Example testing framework
  },
```
2. Don't forget to change the test above into `"test":"jest"`